apnsworker/src/test/scala/apnsworker/payload/ApnsPayloadSpec.scala

```diff
@@ -0,0 +1,270 @@
+        |   "uniqueIdentifier":"068b3d2b-dc9d-482b-a1c9-bd0f5dd8ebd7"
+        |}""".stripMargin
+    )
+  }
+
+  trait MatchStatusNotificationScope extends NotificationScope {
+    val notification = models.FootballMatchStatusNotification(
+      id = UUID.fromString("068b3d2b-dc9d-482b-a1c9-bd0f5dd8ebd7"),
+      sender = "some-sender",
+      title = "Some live event",
+      message = "normal message",
+      thumbnailUrl = None,
+      awayTeamName = "Burnley",
+      awayTeamScore = 1,
+      awayTeamMessage = "Andre Gray 90 +2:41 Pen",
+      awayTeamId = "70",
+      homeTeamName = "Arsenal",
+      homeTeamScore = 2,
+      homeTeamMessage = "Shkodran Mustafi 59\nAlexis Sanchez 90 +7:14 Pen",
+      homeTeamId = "1006",
+      competitionName = Some("Premier League"),
+      venue = Some("Emirates Stadium"),
+      matchId = "1000",
+      matchInfoUri = new URI("https://mobile.guardianapis.com/sport/football/match-info/3955232"),
+      articleUri = Some(new URI("https://mobile.guardianapis.com/items/some-liveblog")),
+      importance = Major,
+      topic = List.empty,
+      matchStatus = "P",
+      eventId = "1000",
+      debug = false
+    )
+
+    val expected = Some(
+      """{
+        |   "t":"football-match-status",
+        |   "aps":{
+        |      "alert":{
+        |         "body":"normal message",
+        |         "title":"Some live event"
+        |      },
+        |      "sound":"default",
+        |      "category":"football-match",
+        |      "mutable-content":1
+        |   },
+        |   "provider":"Guardian",
+        |   "notificationType":"football-match-status",
+        |   "uniqueIdentifier":"068b3d2b-dc9d-482b-a1c9-bd0f5dd8ebd7",
+        |   "footballMatch":{
+        |      "articleUri":"https://mobile.guardianapis.com/items/some-liveblog",
+        |      "awayTeamName":"Burnley",
+        |      "awayTeamScore":"1",
+        |      "currentMinute":"",
+        |      "matchInfoUri":"https://mobile.guardianapis.com/sport/football/match-info/3955232",
+        |      "homeTeamId":"1006",
+        |      "homeTeamScore":"2",
+        |      "homeTeamText":"Shkodran Mustafi 59\nAlexis Sanchez 90 +7:14 Pen",
+        |      "awayTeamId":"70",
+        |      "awayTeamText":"Andre Gray 90 +2:41 Pen",
+        |      "homeTeamName":"Arsenal",
+        |      "matchStatus":"P",
+        |      "competitionName":"Premier League",
+        |      "mapiUrl":"https://mobile.guardianapis.com/sport/football/match-info/3955232",
+        |      "venue":"Emirates Stadium",
+        |      "matchId":"1000"
+        |   }
+        |}""".stripMargin
+    )
+  }
+
+  trait NewsstandNotificationScope extends NotificationScope {
+    val notification = models.NewsstandShardNotification(UUID.randomUUID(), 3)
+
+    val expected = Some(
+      """
+        |{
+        |   "aps":{
+        |      "content-available":1
+        |   }
+        |}""".stripMargin
+    )
+  }
+}
```

Comment:
Those tests are a bit meaningless for now because I had no real source of truth for the expected json. However it would be very useful if/when changing the `APNSPayload` logic